climada/util/test/test_coordinates.py

```diff
@@ -295,7 +296,27 @@ def test_mapping_grid2flattened(self):
         self.assertEqual(out, 39)
         with self.assertRaises(ValueError):
             mapping_grid2flattened(4, 7, matrix.shape)
-        
+
+    def test_assign_coordinates(self):
+        """Test assign_coordinates function"""
+        coords = np.array([(0.2, 2), (0, 0), (0, 2), (2.1, 3), (1, 1), (-1, 1)])
+        coords_to_assign = np.array([(2.1, 3), (0, 0), (0, 2), (0.9, 1.0)])
+        expected_results = [
+            # test with different thresholds
+            (100, [2, 1, 2, 0, 3, -1]),
```

Comment:
I am a bit confused. Why is the distance from `(-1,1)` to `(0,0)` larger than `100` and `20`?

Reply:
Because, we are in a lat-lon-coordinate system and the threshold distance is in km.

Reply:
I suggested to add this to the defaults, i.e., 100km (see suggestions)